src/domain/header/_accessibilityMenu.scss

```diff
@@ -0,0 +1,33 @@
+.app-AccessibilityMenu {
+    padding: 15px 30px;
+    width: 420px;
+    overflow: scroll;
```

Comment:
```suggestion
    overflow: hidden;
```

Otherwise the menu will have two scrollbars.

Reply:
Good catch!